docs/latest/getting-started/running-locally.md

```diff
@@ -39,13 +39,20 @@ If you want to change the port or host, modify the config bag of the `start()`
 call in `main.ts` to include an explicit port number:
 
 ```js main.ts
```

Comment:
I know this isn't from you, but we no longer modify `main.ts` to do this. The correct spot is now in `fresh.config.ts`.  (Otherwise these changes wouldn't take place when running in dev mode.) Additionally, setting `port` and `hostname` directly are deprecated. Please see `FreshConfig` for details.

I guess the correct thing would be something like:
```ts
export default defineConfig({
  server: {
    port: 3000,
    hostname: "0.0.0.0",
  },
});
```

Reply:
I wasn't sure on how to properly update the docs properly. This helped! Thank you for the hint!